doc/man1/genpkey.pod

```diff
@@ -63,13 +63,27 @@ options.
 
 Public key algorithm to use such as RSA, DSA or DH. If used this option must
 precede any B<-pkeyopt> options. The options B<-paramfile> and B<-algorithm>
-are mutually exclusive.
+are mutually exclusive. Engines may add algorithms in addition to the standard
+built-in ones.
+
+Valid built-in algorithm names for private key generation are RSA, RSA-PSS, EC,
+X25519, X448, ED25519 and ED448.
+
+Valid built-in algorithm names for parameter generation (see the B<-genparam>
+option) are DH, DSA and EC.
+
+Note that the algorithm name X9.42 DH may be used as a synonym for the DH
+algorithm. These are identical in every respect and do not indicate the type of
```

Comment:
in every respect ? or every aspect ?

Reply:
"in every aspect" just sounds wrong to me in this context.

Reply:
Just get rid of the "These are identical..." sentence, if it's an alias then they are the same.

Reply:
I kind of wanted emphasise that these don't work as you might expect from the name. The natural inclination is to assume that you would use the "DH" name to generate "DH" (i.e. PKCS3) parameters, and the "X9.42 DH" name to generate X9.42 DH parameters. At least that was my assumption until I looked at how this actually works. I'd rather not drop that sentence.

Reply:
You should drop "in every respect" it's redundant especially immediately after the word "identical"

Reply:
Ok. I did that.